frontends/web/src/routes/new-settings/bb02-settings.tsx

```diff
@@ -106,7 +106,6 @@ const Content = ({ deviceID }: TProps) => {
         {
           versionInfo ?
```

Comment:
I believe we're not missing anything here.

The `versionInfo.canUpgrade` is inside the `FirmwareSetting` itself.  There's a condition `canUpgrade` which is the `versionInfo.canUpgrade`. We can get it in that component since `versionInfo` gets passed. 

https://github.com/digitalbitbox/bitbox-wallet-app/blob/656a1aa3fb1aabf19f6d0ee9d7c96259ce2a0b1c/frontends/web/src/routes/new-settings/components/device-settings/firmware-setting.tsx#LL44C9-L44C19

I think this suffices?